ironfish/src/assert.ts

```diff
@@ -42,13 +42,13 @@ export class Assert {
   }
 
   static isTrue(x: boolean, message?: string): asserts x is true {
-    if (x === false) {
+    if (!x) {
```

Comment:
carefull with replacing `===`

```
let x = null

(!x)
true

(x === false)
false
```

Reply:
Yeah but based on the parameters which seem to be "boolean" that shoudn't happen. I see a strict "boolean" there, not a "boolean | null"

Reply:
right, that's true, TS type checking can protect us, nvm :)